src/utils.ts

```diff
@@ -92,3 +95,34 @@ export function getPrescriptionTableEndpoint(
 ) {
   return `${fhirBaseUrl}/Encounter?_getpagesoffset=${pageOffset}&_count=${pageSize}&subject.name=${patientSearchTerm}&_revinclude=MedicationRequest:encounter&_revinclude:iterate=MedicationDispense:prescription&_has:MedicationRequest:encounter:status=${status}&_sort=-date&_tag=http%3A%2F%2Ffhir.openmrs.org%2Fext%2Fencounter-tag%7Cencounter`;
 }
+
+export function getMedicationsByConceptEndpoint(conceptUuid: string) {
+  return `${fhirBaseUrl}/Medication?code=${conceptUuid}`;
+}
+
+export function getConceptUuidCoding(codings: Coding[]) {
+  // the concept uuid code is always the one without a system
+  return codings ? codings.find((c) => !("system" in c))?.code : null;
+}
+
+export function getOpenMRSMedicineDrugName(medication: Medication) {
```

Comment:
@ibacher  I based this on how the OpenMRS drug name extension appeared to be populated in the FHIR2 module for medications, appreciate you taking a look to see if there are any flaws in my logic or any reason this isn't a standard I should be relying on.  (It's working in my testing).